doc/doc_homepage.md

```diff
@@ -72,9 +72,7 @@ Star Spoon on Github: [https://github.com/INRIA/spoon](https://github.com/INRIA/
 
 ### Jar file
 
-[v{{site.spoon_release}} JAR](https://gforge.inria.fr/frs/?group_id=73) - [Javadoc](http://spoon.gforge.inria.fr/mvnsites/spoon-core/apidocs)
-
-The source code to the Spoon and this website is [available on GitHub](https://github.com/INRIA/spoon).
+Latest version: <https://search.maven.org/remote_content?g=fr.inria.gforge.spoon&a=spoon-core&v=LATEST&c=jar-with-dependencies> - [Javadoc](http://spoon.gforge.inria.fr/mvnsites/spoon-core/apidocs)
```

Comment:
> http://spoon.gforge.inria.fr/mvnsites/spoon-core/apidocs

returns 404 Not found

Reply:
Yep we got an issue about that #2790. I just checked the job https://ci.inria.fr/sos/job/Website%20Deployer/1069/console but it seems the upload worked:

> [INFO] --- maven-site-plugin:3.3:deploy (default-cli) @ spoon-core ---
[INFO] Parent project loaded from repository: org.sonatype.oss:oss-parent:pom:7
Using private key: /builds/.ssh/id_rsa_gforge_spoon_bot
scp://scm.gforge.inria.fr/home/groups/spoon/htdocs/mvnsites/spoon-core/ - Session: Opened  
[INFO] Pushing /builds/workspace/Website Deployer/temp-spoon-clone/target/site
[INFO]    >>> to scp://scm.gforge.inria.fr/home/groups/spoon/htdocs/mvnsites/spoon-core/spoon-core
Executing command: mkdir -p "/home/groups/spoon/htdocs/mvnsites/spoon-core/spoon-core"
Executing command: mkdir -p "/home/groups/spoon/htdocs/mvnsites/spoon-core/spoon-core"
Executing: mkdir -p "/home/groups/spoon/htdocs/mvnsites/spoon-core/spoon-core"
Stdout results:
Stderr results:
Executing command: mkdir -p "/home/groups/spoon/htdocs/mvnsites/spoon-core/spoon-core"
Executing command: mkdir -p "/home/groups/spoon/htdocs/mvnsites/spoon-core/spoon-core"
Executing: mkdir -p "/home/groups/spoon/htdocs/mvnsites/spoon-core/spoon-core"
Stdout results:
Stderr results:
Executing command: scp -t "/home/groups/spoon/htdocs/mvnsites/spoon-core/spoon-core/wagon2724432026535619549.zip"
Uploading: spoon-core/wagon2724432026535619549.zip to scp://scm.gforge.inria.fr/home/groups/spoon/htdocs/mvnsites/spoon-core/
#########################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################################
Transfer finished. 9931867 bytes copied in 0.242 seconds

Reply:
fixed with correct url: http://spoon.gforge.inria.fr/mvnsites/spoon-core/spoon-core/apidocs/index.html